source/developer/developer-setup.md

```diff
@@ -90,6 +90,14 @@ Any issues? Please let us know on our forums at: http://forum.mattermost.org
 
 ### Archlinux ###
 
+#### Build and install with AUR ####
```

Comment:
Also, since this is unoffical, please move this section below the heading `#### Build and install manually`